Makefile

```diff
@@ -55,3 +55,5 @@ testdata:
 login:
 	target=login npm run compile
 
+instrument_manager:
+	target=instrument_manager npm run compile
```

Comment:
is this a leftover or are we adding this for every module now?

Reply:
I've been adding it for modules when I work on them because it makes life easier but haven't systematically added every module.